pkg/register/websocket.go

```diff
@@ -33,14 +33,15 @@ const (
 	MsgReady
 	MsgSmbios
 	MsgLabels
-	MsgGet                     // v0.5.0
-	MsgVersion                 // v1.1.0
-	MsgSystemData              // v1.1.1
-	MsgConfig                  // v1.1.1
-	MsgError                   // v1.1.1
-	MsgAnnotations             // v1.1.4
-	MsgUpdate                  // v1.2.6
-	MsgLast        = MsgUpdate // MsgLast must point to the last message
+	MsgGet                            // v0.5.0
+	MsgVersion                        // v1.1.0
+	MsgSystemData                     // v1.1.1
+	MsgConfig                         // v1.1.1
+	MsgError                          // v1.1.1
+	MsgAnnotations                    // v1.1.4
+	MsgUpdate                         // v1.2.6
+	MsgSystemDataNG                   // v1.6.0
```

Comment:
What about something that makes more explicit it is superseeding MsgSystemData? something like MsgSystemDataV2